taichi/runtime/program_impls/llvm/llvm_program.cpp

```diff
@@ -90,12 +90,12 @@ void LlvmProgramImpl::materialize_snode_tree(SNodeTree *tree,
 std::unique_ptr<AotModuleBuilder> LlvmProgramImpl::make_aot_module_builder(
     const DeviceCapabilityConfig &caps) {
   if (config->arch == Arch::x64 || config->arch == Arch::arm64) {
-    return std::make_unique<cpu::AotModuleBuilderImpl>(config, this);
+    return std::make_unique<cpu::AotModuleBuilderImpl>(*config, this);
```

Comment:
ditto

Reply:
Thanks for your suggestion. I will change it in next PR.